docs/Plugins.rst

```diff
@@ -173,6 +176,8 @@ Usage and related commands:
                 Only useful where (eg) you abandoned with the fort revealed
                 and no longer want the data.
 
+.. _showmood:
```

Comment:
Not that I know of - headers are implicit anchors, but only within the document.  Since we need cross-document internal links, we need the anchors.